src/base/anim/interpolated.h

```diff
@@ -41,6 +41,20 @@ template <typename Type> class Weighted
 	[[nodiscard]] bool hasValue() const { return weight > 0.0; }
 };
 
+enum class InterpolateIndex : bool {
+	first = false,
+	secondIfExists = true
```

Comment:
Reviewing the usage of this enum, it seams to me, that the correct naming would be `first` and `second`, because `secondIfExists` can refer to the first value only in the get() method. So I think the get() method should have an adequate name instead, like `get_or_first()` or something like that.

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/521/commits/3402a37a53b3b25302b9f72200da369ede94aeae